WebHostLib/static/assets/gameInfo/en_Final Fantasy.md

```diff
@@ -1,21 +1,26 @@
 # Final Fantasy 1 (NES)
 
 ## Where is the settings page?
-Unlike most games on Archipelago.gg, Final Fantasy 1's settings are controlled entirely by the original randomzier.
-You can find an exhaustive list of documented settings [here](https://finalfantasyrandomizer.com/)
+
+Unlike most games on Archipelago.gg, Final Fantasy 1's settings are controlled entirely by the original randomzier. You
+can find an exhaustive list of documented settings on the FFR
+website: [FF1R Website](https://finalfantasyrandomizer.com/)
 
 ## What does randomization do to this game?
-A better questions is what isn't randomized at this point. Enemies stats and spell, character spells, shop inventory
-and boss stats and spells are all commonly randomized. Unlike most other randomizers it is also most standard to shuffle
-progression items and non-progression items into separate pools and then redistribute them to their respective 
-locations. So ,for example, Princess Sarah may have the CANOE instead of the LUTE; however, she will never have a Heal 
-Pot or some armor. There are plenty of other things that can be randomized on our 
-[main randomizer site](https://finalfantasyrandomizer.com/)
+
+A better questions is what isn't randomized at this point. Enemies stats and spell, character spells, shop inventory and
+boss stats and spells are all commonly randomized. Unlike most other randomizers it is also most standard to shuffle
+progression items and non-progression items into separate pools and then redistribute them to their respective
+locations. So ,for example, Princess Sarah may have the CANOE instead of the LUTE; however, she will never have a Heal
```

Comment:
Comma misplaced. It was an original typo but might as well fix it here so we don't have conflicting PR's